systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaTopicUtils.java

```diff
@@ -107,18 +107,18 @@ public static void waitForKafkaTopicDeletion(String namespaceName, String topicN
     }
 
     public static void waitForKafkaTopicPartitionChange(String namespaceName, String topicName, int partitions) {
-        LOGGER.info("Waiting for KafkaTopic change {}", topicName);
-        TestUtils.waitFor("KafkaTopic change " + topicName, Constants.POLL_INTERVAL_FOR_RESOURCE_READINESS, Constants.GLOBAL_TIMEOUT,
+        LOGGER.info("Waiting for KafkaTopic: {} to change", topicName);
```

Comment:
Also maybe add there Namespace? `KafkaTopic: {}/{} to change` (applies also for other occurrences)